app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -105,53 +105,78 @@ $(document).ready(function() {
     }
 
     var bountyAmount = parseInt($('input[name=valueInToken]').val(), 10);
-    var ethAmount = isETH ? amount : 0;
-    var bountyId = $('input[name=standardBountiesId]').val();
-    var fromAddress = $('input[name=bountyOwnerAddress]').val();
 
-    var errormsg = undefined;
-
-    if (bountyAmount == 0 || open == false) {
-      errormsg = 'No active funded issue found at this address. Are you sure this is an active funded issue?';
-    }
-    if (fromAddress != web3.eth.coinbase) {
-      errormsg = 'Only the address that submitted this funded issue may increase the payout.';
-    }
-
-    if (errormsg) {
-      _alert({ message: errormsg });
-      unloading_button($('#submitBounty'));
-      return;
-    }
-
-    function approveSuccessCallback() {
-      bounty.increasePayout(
-        bountyId,
-        bountyAmount + amount,
-        amount,
-        {
-          from: account,
-          value: ethAmount,
-          gasPrice: web3.toHex($('#gasPrice').val()) * Math.pow(10, 9)
-        },
-        web3Callback
-      );
-    }
+    // Use DB to determine which networks each bounty is on.
+    var uri = '/api/v0.1/bounties/?github_url=' + issueURL;
+
+    $.get(uri, function(results, status) {
```

Comment:
sometimes were using the API as the source of truth and sometimes we're using the smart contract.. is there a rule you followed to decide which to use where?

Reply:
In this case, have to use the DB to resolve which of the two blockchains the issue is backed on. I only used the blockchain as truth for expiration-date and status.

Reply:
ok makes sense